platform/darwin/src/MGLNetworkConfiguration.h

```diff
@@ -32,6 +32,14 @@ MGL_EXPORT
  */
 @property (atomic, strong, null_resettable) NSURLSessionConfiguration *sessionConfiguration;
 
+/**
+A Boolean value indicating whether the current `NSURLSessionConfiguration` stops
+making network requests.
```

Comment:
Let’s also add some details about why this property might be useful — e.g., because it can allow developers to rely solely on pre-cached resources. We might also want to update the general offline docs (here in the SDK but also possibly in examples or tutorials) to mention this new feature.